chicken-sloths/bangazon-api-sprint1
# Create Product Table

# Description
Add's an exported anonymous function to makeProductTable.js which creates the product table using the products.json.

## Related Ticket(s)
Finished Issue #41 (Create Product Table)

## Expected Behavior
When you require in makeProductTable.js it should export a function that when you call creates a Products database with pertinent information.

## Steps to Test Solution

1. Run: cd db/products/
1. Run:  node
1. Run: const createProduct = require('./makeProductsTable');
1. Run: createProduct();
1. Open api-sprint.sqlite located in the db/ folder in your sql browser of choice and check if a Products table has been created.
